### ISSUE TITLE
French Détails section of Recall contains English words

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#95]
URL: https://hcdev.openplus.ca/fr/avis-rappel/test-rappel-salades-chou-frise-marque-curation-foods-et-salades-hachees-pretes-servir
User Agent: Chrome
Language: French

Issue Description: the "Détails" section of the recalls page contains English labels. 
Steps to reproduce:

1. Navigate to the URL
2. Scroll to the "Renseignement supplémentaires" section and expand "Détails" section. 
3. Original published Date, the Issuing organization as well as Identification number labels are in EN on the FR page. 
![image](https://user-images.githubusercontent.com/65728700/132499247-f7c39b86-8d5d-438a-8e7a-a06ddc731da1.png)

Expected results:
These should be translated. if not then, you have to wrap the text within a lang attribute to ensure that the words are properly said by assistive technologies.

**Comment by ndolter** [comment]
Validated that the text has been translated.
![image](https://user-images.githubusercontent.com/68117281/135870125-2da76d86-4cc0-4d5f-ac45-1dcbae76de5e.png)
